src/test/kotlin/bsu/cc/xlsx_parser/XlsxHighlightingTest.kt

```diff
@@ -0,0 +1,113 @@
+package bsu.cc.xlsx_parser
+
+import bsu.cc.parser.*
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+import org.apache.poi.ss.usermodel.IndexedColors
+import org.apache.poi.ss.usermodel.Workbook
+import java.io.FileOutputStream
+import java.lang.IllegalArgumentException
+
+class XlsxHighlightingTest : WordSpec() {
+    private val basePath = "src/test/resources/xlsx/"
+    private val DEFAULT_SHEET_INDEX = 0
+
+    init {
+        "XLSX highlighting" should {
+
+            val happyPathPath = basePath + "HappyPath.xlsx"
+            val unevenRowLengthPath = basePath + "UnevenRowLengths.xlsx"
+            val tempOutputPath = basePath + "HighlightingTemp.xlsx"
+
+            fun verifyHighlighting(filePath: String, sheetIndex: Int, rowIndex: Int, cellIndices: List<Int>, color: IndexedColors) {
+                val workbook = readWorkbook(filePath)
+                val row = workbook.getSheetAt(sheetIndex).getRow(rowIndex);
+                (0 until ((cellIndices.max() ?: throw IllegalArgumentException("Empty cell indices")) + 1)).filter { index -> cellIndices.contains(index) }.forEach { index ->
+                    row.getCell(index).cellStyle.fillForegroundXSSFColor.index shouldBe color.index
```

Comment:
I do like how kotlin automatically converts getters and setters into properties.